.gitignore

```diff
@@ -13,3 +13,5 @@ node_modules
 dist
 docs/docs
 .DS_Store
+*.swp
+cscope.*
```

Comment:
@unclejack yes, you are right. It is needed to avoid showing my ide and tags files in git status. I just slipped it in with this PR to avoid having to create a separate issue. Let me remove it and create a separate PR.